src/py2spack/conversion_tools.py

```diff
@@ -64,19 +62,7 @@ def requirement(self) -> str | None:
 
 
 # TODO @davhofer: integrate this into a datastructure, omit mutable global state  # noqa: TD003
```

Comment:
`TD003` is too annoying. Just add the error to the list of ignored errors in the ruff section of the pyproject.toml file
ignore = [
  "TD003",  # [missing-todo-link]
]